package.json

```diff
@@ -11,25 +11,25 @@
     "@types/sinon": "^10.0.13",
     "@typescript-eslint/eslint-plugin": "^5.47.1",
     "@typescript-eslint/parser": "^5.47.1",
-    "c8": "^7.11.3",
-    "commitizen": "^4.2.5",
+    "c8": "^7.12.0",
+    "commitizen": "^4.2.6",
     "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^8.20.0",
+    "eslint": "^8.31.0",
     "eslint-config-prettier": "^8.5.0",
     "husky": "^8.0.2",
     "lint-staged": "^13.1.0",
     "prettier": "^2.8.1",
-    "sinon": "^12.0.1",
+    "sinon": "^15.0.1",
     "tsx": "^3.8.0",
-    "typescript": "^4.7.4",
+    "typescript": "^4.9.4",
     "uvu": "^0.5.6"
   },
   "scripts": {
     "build": "pnpm --recursive --parallel build",
     "changeset": "changeset && pnpm install",
     "clean": "pnpm --recursive --parallel clean",
     "clean:nodemodules": "find . -type d -name \"node_modules\" -exec rm -fr {} +",
-    "dev": "pnpm --recursive --parallel dev",
+    "dev": "pnpm --recursive --parallel --filter !./packages/plugin-*/web dev",
```

Comment:
This let's us add new plugins and `pnpm dev` will automatically run for their `cli` directory.

I think the solution to make it also work for plugin `web` directories is to use dynamic ports for `plugin-*/web` SvelteKit apps. Start at 4000 for the first plugin, then increment for each additional plugin.

The challenge is to make it deterministic because we use `--parallel` here. We could hard-code ports, start with 4000 for example and increment by one for each new plugin we add. As long as we coordinate when multiple new plugins are being worked on, it's likely the simplest solution.

What do you think?

Reply:
At least for the cli plugins this makes sense. When working on multiple web-plugins, I don't have an issue starting them manually. I think it's quite hard to determine which web-plugins to start and which not, as this may even depend on the coding style of the dev 🤔

Reply:
Agreed, too much effort for not enough benefits.